src/modules/LeaderboardAPI.js

```diff
@@ -41,9 +41,7 @@ const showScores = async (userData) => {
   scoresDisplay.innerHTML = '';
   sortedData.forEach((element) => {
     const score = `
-    <div>
-    <p>${element.user}:<span>${element.score}</span></p>
-    </div>
+    <li>${element.user}:<span class="space">${element.score}</span></li>
     `;
     scoresDisplay.innerHTML += score;
```

Comment:
Please ensure that the app style and its layout look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).

Your Implementation | wireframe Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/162587969-d538b7d8-9777-48b3-b752-36d7bf8fe344.png) | ![image](https://user-images.githubusercontent.com/10332499/162587930-9a2811a9-36a9-4716-ae2d-ec5d9245dce1.png)

Reply:
@micheaol thank you, app style and layout updated. 
![e45](https://user-images.githubusercontent.com/40554384/162588354-19fe9021-69c9-48c5-8c30-0ba07463493c.PNG)